packages/core/src/components/upload/upload.tsx

```diff
@@ -267,14 +267,14 @@ export class Upload {
               type="file"
               class="upload-browser"
               id="upload-browser"
+              tabindex="-1"
               onChange={(e) => {
                 this.fileChangeEvent(e);
               }}
               accept={this.accept}
               disabled={disabled}
             />
             <ix-button
```

Comment:
We should check what can be done in terms of a11y here. At least `aria-disabled` should be set.